File: scripts/reset-password.go
package main

import (
	"fmt"
	"log"

	"golang.org/x/crypto/bcrypt"
)

func main() {
	password := "pri@expo"

	hash, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
	if err != nil {
		log.Fatal("Error generating hash:", err)
	}

	fmt.Println("Bcrypt hash for password 'pri@expo':")
	fmt.Println(string(hash))
}

File: scripts/reset-password.js
const bcrypt = require('bcrypt');

const password = 'pri@expo';
const saltRounds = 10;

bcrypt.hash(password, saltRounds, (err, hash) => {
  if (err) {
    console.error('Error generating hash:', err);
    process.exit(1);
  }
  console.log('Bcrypt hash for password "pri@expo":');
  console.log(hash);
});
